### PR TITLE
Fixes overlay interactions

### PR DESCRIPTION
**Pull request by javierarce**
This PR… 

- Fixes a bug that closed the overlay when enter was pressed.

### CODE REVIEW

**Comment by xavijam** [comment]
ook!

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3010/))

**Comment by javierarce** [comment]
@xavijam: can you review this, please :)

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3016/))

File: lib/assets/javascripts/cartodb/table/overlays/text.js [hunk 1]
```diff
@@ -128,7 +128,7 @@ cdb.admin.overlays.Text = cdb.geo.ui.Text.extend({
       }
     }
 
-    if (e.keyCode === 27) {
+    if (e.keyCode === 27) { // esc key
```
Comment (viddo) [inline lib/assets/javascripts/cartodb/table/overlays/text.js]: For curiosity, I saw that @alonsogarciapablo used `$.ui.keycode.ESCAPE` (I didn't know about these before at least) on the inline description/tags edit, is there some reason for why we use one or another?

Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/overlays/text.js]: We should unify with a decision, because I remember to have used ```e.which``` as well.

**Comment by xavijam** [comment]
:+1:

File: lib/assets/javascripts/cartodb/table/overlays/text.js (hunk 1 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb/table/overlays/text.js]: Oh, nice, I didn't know of $.ui.keycode.ESCAPE. I'll use that, it's more explicit.

Comment (javierarce) [inline lib/assets/javascripts/cartodb/table/overlays/text.js]: Oh, wait, that requires jquery ui.

Comment (viddo) [inline lib/assets/javascripts/cartodb/table/overlays/text.js]: Aha, which we happen to load as a vendor dependency: https://github.com/CartoDB/cartodb/blob/master/lib/build/files/js_files.js#L13 are we planning on removing it or is there some reason against using it? I think we should decide on one or other (and have it clear why). cc @javisantana

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3017/))

File: lib/assets/javascripts/cartodb/table/overlays/text.js (hunk 1 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb/table/overlays/text.js]: I don't think we can remove it easily right now. We are using draggable, sortable and droppable plugins right now.

**Pull request merged**